modules/sno-du-enabling-workload-partitioning.adoc

```diff
@@ -52,9 +52,11 @@ spec:
 [crio.runtime.workloads.management]
 activation_annotation = "target.workload.openshift.io/management"
 annotation_prefix = "resources.workload.openshift.io"
-resources = { "cpushares" = 0, "cpuset" = "0-1,52-53" } <1>
+[crio.runtime.workloads.management.resources]
+cpushares = 0
+cpuset = "0-1, 52-53" <1>
 ----
-<1> The `CPUs` value varies based on the installation.
+<1> The `cpuset` value varies based on the installation.
 +
 If Hyper-Threading is enabled, specify both threads for each core. The `CPUs` value must match the reserved CPU set specified in the performance profile.
```

Comment:
Does this also need a chance `CPUs` -> `cpuset`? Also, I think we should highlight the word `reserved` if we mean to reference the `reserved` field in the performance profile so it's clear we mean that field.